bank_statement_journal_items/models/bank_statement_items.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models
+
+class BankStatementItems(models.Model):
+
+	_inherit="account.bank.statement"
+
+	def button_journal_items(self):
+		return {
+			'name': 'Journal Items',
+			'view_mode': 'tree,form',
+			'res_model': 'account.move.line',
+			'views':[
+				(self.env.ref("account.view_move_line_tree").id, 'tree'),
+				(self.env.ref("account.view_move_line_form").id, 'form')
+			],
+			'type': 'ir.actions.act_window',
+			'domain': [('id', 'in', self.mapped('line_ids').ids)],
```

Comment:
```suggestion
			'domain': [('id', 'in', self.mapped('move_line_ids').ids)],
```
:warning: 
`line_ids` here will give you `account.bank.statement.line` records: https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_bank_statement.py#L196
Here we want to display `account.move.line`, so you must use `move_line_ids`: https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_bank_statement.py#L197